frame/uniques/src/lib.rs

```diff
@@ -122,6 +130,9 @@ pub mod pallet {
 		#[pallet::constant]
 		type ValueLimit: Get<u32>;
 
+		/// A set of helper functions for benchmarking.
+		type Helper: BenchmarkHelper<Self::ClassId, Self::InstanceId>;
```

Comment:
Could we make this a `#[cfg(feature = "runtime-benchmarks")]` as it is only needed for benchmarking?

Reply:
Yes. The reason I didn't it because it would realistically just mean a few extra lines of code in the codebase and in every project's runtime file which used this pallet.

Reply:
Well, if you want to go extra sophisticated you could add a `Helper` for `not(feature=runtime…)` which is just a `core::any::Any`.  
Then a normal runtime could still configure it with `= ()` without extra `feature` lines.

Reply:
(Done)